packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolylineController.h

```diff
@@ -22,4 +25,5 @@
 - (void)removePolylineWithIdentifiers:(NSArray *)identifiers;
 - (void)didTapPolylineWithIdentifier:(NSString *)identifier;
 - (bool)hasPolylineWithIdentifier:(NSString *)identifier;
++ (GMSMutablePath *)getPath:(NSDictionary *)polyline;
```

Comment:
Obj-C methods do not start with `get`, and they should name their parameters in the signature. This should be called `pathForPolyline`.